build.cmd

```diff
@@ -1,5 +1,12 @@
 @echo off
+.paket\paket.bootstrapper.exe
```

Comment:
`.paket\paket.bootstrapper.exe 3.26.3`

That way it'll always download the same Paket version (usable in case new versions will break something). I think that it's better to explicitly specify exact versions of every dependency; isn't it one of the core ideas behind `dotnet restore` stuff?